src/pycram/process_modules/pr2_process_modules.py

```diff
@@ -1,3 +1,5 @@
+import actionlib
```

Comment:
Please remove this Import since it will fail in ROS2, use the Abstraction in pycram.ros instead